cssom-1/Overview.bs

```diff
@@ -2779,19 +2782,27 @@ steps:
   versions. That is, both <code>:before</code> and <code>::before</code> should
   match above.
 
+ <li>Let <var>decls</var> be an empty list of <a>CSS declarations</a>.
+ <li>If <var>elt</var> is <a>connected</a>, part of the <a>flat tree</a>,
+  and its <a>shadow-including root</a>'s <a>browsing context</a> either doesn't have a
+  <a>browsing context container</a>, or that <a>browsing context container</a>
+  is <a>being rendered</a>,
+  let <var>decls</var> be a list of all longhand properties that are
```

Comment:
You want to use set here, since you're overwriting something. See https://infra.spec.whatwg.org/. (Older specifications are inconsistent with this unfortunately.)